python_backend/api.py

```
@@ -36,14 +130,278 @@ async def root():
+                status_code=500,
+                detail="Output file was not created"
+            )
+
+        return FileResponse(
+            output_path,
+            media_type="application/vnd.openxmlformats-officedocument.presentationml.presentation",
+            filename="translated.pptx",
+        )
+
+    except json.JSONDecodeError as e:
+        logger.error(f"Invalid translations JSON: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)